ui/src/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js

```diff
@@ -44,6 +44,15 @@ class LoanRequestListEntry extends Component {
               <Item.Meta>
                 <DocumentAuthors metadata={loan.metadata.document} />
                 Requested on {toShortDate(loan.metadata.request_start_date)}
+                <br />
+                Valid until {toShortDate(loan.metadata.request_expire_date)}
```

Comment:
Do we have a task to implement this? :)
Ping @kprzerwa maybe

Reply:
as I remember we didn't have a task but it makes sense to have this information here. @topless as you have made some changes in the looks, can we see a screen or demo?